docs/getting-started.md

```diff
@@ -4,26 +4,192 @@
+
+---
+
+#### Download and Install Regen Ledger
+
+Install git and build-essentials for ubuntu
+```sh
+sudo apt update
+```
+```sh
+sudo apt install git build-essential jq -y
+```
+
+Download go and extract tarball
+```sh
+wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
+```
+```sh
+tar -xvf go1.15.2.linux-amd64.tar.gz
+```
+```sh
+sudo mv go /usr/local
+```
+
+Add go to local path
+```sh
+echo "" >> ~/.profile
+```
+```sh
+echo 'export GOPATH=$HOME/go' >> ~/.profile
+```
+```sh
+echo 'export GOROOT=/usr/local/go' >> ~/.profile
+```
+```sh
+echo 'export GOBIN=$GOPATH/bin' >> ~/.profile
+```
+```sh
+echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> ~/.profile
+```
+
+```sh
+. ~/.profile
+```
+
+Check to make sure you have the correct version of installed **version 1.15** or higher
+```sh
+go version
+```
+
+Next, we download the official Regen ledger github repository
+```sh
+cd $HOME
+```
+```sh
+git clone https://github.com/regen-network/regen-ledger $GOPATH/src/github.com/regen-network/regen-ledger
+```
+```sh
+cd $GOPATH/src/github.com/regen-network/regen-ledger
+```
+
+Run `make install` to build the code.
+```sh
+make install
+```
+
+Once installed the `regen` binary serves as both the blockchain daemon, as well as
 the command line blockchain client.
 
+## Creating Keys
+
+Run this command replacing `[moniker]` with the name of your key(you get to choose this). 
+
+::: **IMPORTANT!**
+
+**Save the information including the mnemonic after running the following command. Without this you will not be able to recover your account!**
+
+:::
+
+```sh
+regen keys add [moniker]
+```
+
+During this process you will need to create a password to secure your private keys. **DO NOT LOSE PASSWORD**
+
+## Setting up Cosmovisor
+
+According to the [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor) `cosmovisor` is a small process manager around Cosmos SDK binaries that monitors the governance module via stdout to see if there's a chain upgrade proposal coming in. If it sees a proposal that gets approved it can be run manually or automatically to download the new code, stop the node, run the migration script, replace the node binary, and start with the new genesis file.
+
+**The following steps help you get `cosmovisor` set up on your machine.**
+
+Remove cosmos-sdk if there is an existing one
+```sh
+rm -rf $GOPATH/src/github.com/cosmos/cosmos-sdk
+```
+
+Clone the cosmos-sdk repository
+```sh
+git clone https://github.com/cosmos/cosmos-sdk $GOPATH/src/github.com/cosmos/cosmos-sdk
+```
+
+Enter cosmos-sdk and use correct version
+```sh
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+```
+```sh
+git checkout v0.40.0    
```

Comment:
This shouldn't be necessary, folks should be building cosmovisor from the same version that they are building regen ledger most of the time.

I think ppl should just be running `go get` as described in the cosmovisor readme:
https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor#installation

If we point folks to just `go get` cosmovisor and read the readme for usage guidelines, I think we can actually leave out most of this section. I think it would be nicer to point to existing documentation where possible rather that duplicating instructions incase they get updated later.